src/main/kotlin/com/github/quillraven/fleks/world.kt

```diff
@@ -142,6 +139,11 @@ class World(
         componentService = ComponentService(worldCfg.componentFactory)
         entityService = EntityService(worldCfg.entityCapacity, componentService)
         val injectables = worldCfg.injectables
+
+        // Add world to inject object so that component listeners can get it form injectables, too
+        // Set "used" to true to make this injectable not mandatory
+        injectables["World"] = Injectable(this, true)
```

Comment:
I added that because in my example (which I write for Korge) I need the world object in a component listener. Since the listener is just an interface there is now the possibility to add a dependency object for the "World" in the final component listener which is injected automatically :)